pydicom/tests/test_overlay_np.py

```diff
@@ -204,9 +204,13 @@ def test_no_frames(self):
     def test_typing_imports(self, monkeypatch):
```

Comment:
I think this whole test can be removed now, it's easier to just tell coverage to ignore anything in `if TYPE_CHECKING` blocks using the `# pragma: no cover` directive